pandas/core/generic.py

```diff
@@ -3607,6 +3607,10 @@ def head(self, n=5):
         """
         Return the first n rows.
 
+        This function returns rows for the object based on the underlying
+        index. It is useful for quickly testing if your object has the
```

Comment:
Might be good to put this in a separate paragraph: 'It is useful for quickly testing if your object has the right type of data in it.'

Reply:
Hmm I thought that would be a bit too long.